packages/feedback-service/service.ts

```diff
@@ -0,0 +1,84 @@
+import express from 'express';
+import { ApolloServer } from 'apollo-server-express';
+import fs from 'fs';
+import https from 'https';
+import http from 'http';
+import { mergeSchemas } from 'graphql-tools';
+const { ApolloLogExtension } = require('apollo-log');
+import mongoose from 'mongoose';
+
+import gqlSchema from './src/typedef.graphql';
+import { FeedbackResolver as resolver } from './src/resolver';
+
+/* Setting port for the server */
+const port = process.env.PORT || 8080;
+
+const app = express();
+const extensions = [() => new ApolloLogExtension({
+  level: 'info',
+  timestamp: true,
+})];
+
+    /* Configuring Mongoose */
+    mongoose.plugin((schema: any) => { schema.options.usePushEach = true; });
+    mongoose.set('useNewUrlParser', true);
+    mongoose.set('useFindAndModify', false);
+    mongoose.set('useCreateIndex', true);
+    mongoose.set('useUnifiedTopology', true);
+
+    /* Establishing mongodb connection */
+    const dbCredentials = (process.env.DB_USER && process.env.DB_PASSWORD)
+      ? `${process.env.DB_USER}:${process.env.DB_PASSWORD}@`
+      : '';
+    const dbConnection = `mongodb://${dbCredentials}${process.env.DB_PATH}/${process.env.DB_NAME}`;
+
+    mongoose.connect(dbConnection, { useNewUrlParser: true, useCreateIndex: true }).catch(console.error);
+
+    mongoose.connection.on('error', error => {
+      console.error(error);
+    });
```

Comment:
Decrease indent

Reply:
@anjneeksharma I'm wondering how your indentation is changing from what we have in the generator?